[2021-11-01 02:34:04,816] {taskinstance.py:1035} INFO - Dependencies all met for <TaskInstance: my_dag.readtable manual__2021-11-01T02:34:01.208424+00:00 [queued]>
[2021-11-01 02:34:04,825] {taskinstance.py:1035} INFO - Dependencies all met for <TaskInstance: my_dag.readtable manual__2021-11-01T02:34:01.208424+00:00 [queued]>
[2021-11-01 02:34:04,825] {taskinstance.py:1241} INFO - 
--------------------------------------------------------------------------------
[2021-11-01 02:34:04,825] {taskinstance.py:1242} INFO - Starting attempt 1 of 1
[2021-11-01 02:34:04,825] {taskinstance.py:1243} INFO - 
--------------------------------------------------------------------------------
[2021-11-01 02:34:04,838] {taskinstance.py:1262} INFO - Executing <Task(PostgresOperator): readtable> on 2021-11-01 02:34:01.208424+00:00
[2021-11-01 02:34:04,843] {standard_task_runner.py:52} INFO - Started process 2943 to run task
[2021-11-01 02:34:04,846] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'my_dag', 'readtable', 'manual__2021-11-01T02:34:01.208424+00:00', '--job-id', '104', '--raw', '--subdir', 'DAGS_FOLDER/checkModel.py', '--cfg-path', '/tmp/tmpjjyapsnw', '--error-file', '/tmp/tmpygr3boqz']
[2021-11-01 02:34:04,846] {standard_task_runner.py:77} INFO - Job 104: Subtask readtable
[2021-11-01 02:34:04,875] {logging_mixin.py:109} INFO - Running <TaskInstance: my_dag.readtable manual__2021-11-01T02:34:01.208424+00:00 [running]> on host 93c37aa78767
[2021-11-01 02:34:04,912] {taskinstance.py:1429} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=my_dag
AIRFLOW_CTX_TASK_ID=readtable
AIRFLOW_CTX_EXECUTION_DATE=2021-11-01T02:34:01.208424+00:00
AIRFLOW_CTX_DAG_RUN_ID=manual__2021-11-01T02:34:01.208424+00:00
[2021-11-01 02:34:04,918] {base.py:79} INFO - Using connection to: id: postgres_default. Host: 172.23.0.3, Port: 5432, Schema: ***, Login: ***, Password: ***, extra: {}
[2021-11-01 02:34:04,920] {dbapi.py:225} INFO - Running statement: select * from passengers limit 10;, parameters: None
[2021-11-01 02:34:04,921] {dbapi.py:233} INFO - Rows affected: 10
[2021-11-01 02:34:04,930] {taskinstance.py:1280} INFO - Marking task as SUCCESS. dag_id=my_dag, task_id=readtable, execution_date=20211101T023401, start_date=20211101T023404, end_date=20211101T023404
[2021-11-01 02:34:04,977] {local_task_job.py:154} INFO - Task exited with return code 0
[2021-11-01 02:34:05,019] {local_task_job.py:264} INFO - 0 downstream tasks scheduled from follow-on schedule check
